docs/data-binding-api.md

```diff
@@ -0,0 +1,81 @@
+## Summary
```

Comment:
It doesn't have the meta information section.

Reply:
Added with [f4d67c0](https://github.com/ballerina-platform/module-ballerina-websocket/pull/541/commits/f4d67c0ceda94a55d53c8751b4b8764e06673ec5)